src/assets/view/components/XExpEditor.vue

```diff
@@ -493,18 +498,16 @@ export default {
 </script>
 <style scoped>
 .fullscreen {
-  position: fixed;
-  /* inset: 0; */
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  height: 100% !important;
-  resize: none;
-  background: #fff;
-  z-index: 1;
-  margin: 0;
-  padding: 1rem;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 100% !important;
+    resize: none;
+    z-index: 1;
+    margin: 0;
+    padding: 1rem;
```

Comment:
It's better to revert some unrelated changes.

```suggestion
  position: fixed;
  /* inset: 0; */
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  height: 100% !important;
  resize: none;
  z-index: 1;
  margin: 0;
  padding: 1rem;
```